cms/static/sass/bootstrap/studio-main.scss

```diff
@@ -3,7 +3,7 @@
 // -----------------------------
 
 // Bootstrap theme
-@import 'bootstrap/theme';
+@import 'cms/bootstrap/theme';
```

Comment:
Curious about the second "cms" subdirectory.  Does this mean "custom", or specifically "cms-custom", or something else?